nmt_chainer/eval.py

```diff
@@ -276,6 +280,8 @@ def define_parser(parser):
     parser.add_argument("--max_nb_ex", type = int, help = "only use the first MAX_NB_EX examples")
     parser.add_argument("--mb_size", type = int, default= 80, help = "Minibatch size")
     parser.add_argument("--beam_width", type = int, default= 20, help = "beam width")
+    parser.add_argument("--beam_size", type = float, default= 3.0, help = "beam size")
```

Comment:
default value should be None (that is, do not give the default parameter here)
This way, as mentioned above, we do not need consider_beam_size_when_pruning. beam_size == None become conceptually equivalent to consider_beam_size_when_pruning == False